This patch renames the cf-monitor.pid to cf-monitord.pid because
the daemon is called cf-monitord and the init script I wrote needs
a PID in the $DAEMON_NAME.pid format.

All the other PID's are correctly referring to their repective daemon
names, this bug was notified to upstream

--- a/src/env_monitor.c
+++ b/src/env_monitor.c
@@ -339,7 +339,7 @@
    return;
    }
 
-WritePID("cf-monitor.pid");
+WritePID("cf-monitord.pid");
 
 OpenSniffer();
  
